config/collections.go

```diff
@@ -74,6 +74,11 @@ type Search struct {
 	// One or more optional templates that make up the autosuggestions. Uses Go text/template syntax to reference fields.
 	SuggestTemplates []string `yaml:"suggestTemplates,omitempty" json:"suggestTemplates,omitempty"`
 
+	// SQLite WHERE clause to filter features when importing/ETL-ing
+	// (Without the WHERE keyword, only the clause)
+	// +Optional
+	ETLFilter string `yaml:"etlFilter,omitempty" json:"etlFilter,omitempty"`
```

Comment:
Stel voor om er `ETL` van te maken en daarbinnen een `Filter`. Dan kan `SuggestTemplates` ook binnen `ETL` worden geplaatst want die hebben we volgens mij ook alleen in ETL pipeline nodig en niet tijdens zoeken zelf. Dan hebben we ook een kapstok voor meerdere ETL settings.

Reply:
done